signing_service/src/signing_service.py

```diff
@@ -0,0 +1,45 @@
+import argparse
```

Comment:
One thing I noticed today when defining https://github.com/golemfactory/concent-deployment/issues/185. You don't have any top-level runner script. I think that having something simple directly inside the `signing_service/` directory would make it easier to discover how to run it.

Reply:
Do you mean shell script ? If yes, then I have added very simple one.

Reply:
Can be a python script, can be a shell script. I just mean any obvious script at the top level because right now someone who wants to run the application has to go into `src/` and figure out which one should be used.

`run.sh` is OK but maybe `signing_service.sh` would be a better name?

Reply:
:+1:  done